pallets/check-membership/src/tight/mod.rs

```diff
@@ -14,7 +14,10 @@ mod tests;
 /// The pallet's configuration trait.
 /// Notice the explicit tight coupling to the `vec-set` pallet
 pub trait Trait: system::Trait + vec_set::Trait {
-	type Event: From<Event<Self>> + Into<<Self as system::Trait>::Event>;
+	// type Event: From<Event<Self>> + Into<<Self as system::Trait>::Event>;
+	type Event: From<Event<Self>>
+		+ Into<<Self as system::Trait>::Event>
+		+ Into<<Self as vec_set::Trait>::Event>;
```

Comment:
Why is this trait bound necessary?

Reply:
I want to add this mainly from the perspective of code correspondence. Because of the above:pub trait Trait: system::Trait + vec_set::Trait
I don't know the main principle of code, I'm a beginner.
Main purpose: compulsory inspection.

Reply:
test find:
fn deposit_event() = default;  use  Into<<Self as system::Trait>::Event>;
now use Into<<Self as vec_set::Trait>::Event>;  
may no need it. @JoshOrndorff . Look again when you have time.
Close it when you don't need it